packages/core/src/application_ref.ts

```diff
@@ -266,6 +266,24 @@ export interface BootstrapOptions {
    * the change detection will only be trigged once.
    */
   ngZoneEventCoalescing?: boolean;
+
+  /**
+   * Optionally specify coalescing change detections or not.
+   * Consider the following case.
+   *
+   * for (let i = 0; i < 10; i ++) {
+   *   ngZone.run(() => {
+   *     // do something
+   *   });
+   * }
+   *
+   * This case triggers the change detection multiple times.
+   * So with ngZoneCoalescing options, all change detections in one
+   * event loop will only be triggered once, also the change detection
+   * will be executed in requestAnimation just like `markDirty()`.
+   *
+   */
+  ngZoneCoalescing?: boolean;
```

Comment:
I am concerned that this name is too generic. Could we find a name which better reflects this? `ngZoneRunCoalescing` to be consistent with `ngZoneEventCoalescing`?

```suggestion
  ngZoneRunCoalescing?: boolean;
```

Reply:
Yeah, this name is better.